### PR TITLE
CFE-3673: Made separate and complete examples for parse*array functions (master)

### PR DESCRIPTION
**Pull request by nickanderson**
merge together: https://github.com/cfengine/documentation/pull/2455

### CODE REVIEW

**Review by larsewi — APPROVED** [review]
ACK

**Comment by olehermanse** [comment]
@cf-bottom jenkins, please.

**Review by olehermanse — DISMISSED** [review]

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/6618/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/6618/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-6618/

File: examples/parserealarray.cf [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+      "dim"
+        int => parserealarray(
+                                "items",
+				                        "$(table)",
+				                        "\s*#[^\n]*",
+				                        ":",
+				                        "1000",
+				                        "200000"
+      );
```
Comment (olehermanse) [inline examples/parserealarray.cf:42]: I don't like this crazy indentation.

Comment (nickanderson) [reply examples/parserealarray.cf]: fixed tabs to spaces in parse(int|real|string)array.cf

**Review by larsewi — APPROVED** [review]
ACK

**Review by olehermanse — APPROVED** [review]

**Pull request merged**